cmd/dogstatsd/main.go

```diff
@@ -182,8 +184,10 @@ func start(cmd *cobra.Command, args []string) error {
 		tagger.Init()
 	}
 
-	aggregatorInstance := aggregator.InitAggregator(s, hname, "agent")
-	statsd, err := dogstatsd.NewServer(aggregatorInstance.GetBufferedChannels())
+	metricSamplePool := metrics.NewMetricSamplePool(32)
```

Comment:
Because it's not configurable, had you the chance to confirm that `32` is optimal? Maybe a small comment on how this value has been chosen could be useful (or even why it's "good enough").

Reply:
It's more "good enough" than "optimal". The goal is to lower the cost of `sync.Pool` and channels operations. This will divide it by 32 and should be enough for any scenario. I'll add a comment.